client/app/models/scheduleitem.coffee

```diff
@@ -84,11 +80,12 @@ module.exports = class ScheduleItem extends Backbone.Model
 
     # Format a moment to the string format of the model.
     _formatMoment: (m) ->
+
         if @isAllDay()
             s = H.momentToDateString(m)
 
-        else if @isRecurrent() and not @has('timezone')
-            s = moment.tz(m, @get('timezone')).toISOString()
+        else if @isRecurrent()
+            s = H.momentToAmbiguousString(m)
```

Comment:
H, m, s ... These variable names are awful and don't follow the guidelines. Can you fix this? It matters for readibility and maintenability.